packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -223,13 +224,19 @@ describe(`E2E: Web frameworks`, () => {
 		},
 	};
 
-	test.concurrent.each(Object.keys(frameworkTests))(
-		"%s",
-		async (name) => {
-			await runCliWithDeploy(name, frameworkTests[name].testCommitMessage);
-		},
-		{ retry: 3 }
-	);
+	Object.keys(frameworkTests).forEach((framework) => {
+		const skip = frameworkToTest && frameworkToTest !== framework;
+		test.skipIf(skip)(
```

Comment:
How does this code replace `test.concurrent` in a way that preserves the ability to run tests concurrently when we run the tests on non-dependabot PRs?

Reply:
ah no sorry about that it doesn't, I missed that when changing the code... 😓

I'll have a look on if/how we can keep the concurrency 🙇

Reply:
Fixed by adding a `.concurrent` in the `describe` as that has the same effect 🙂  ([docs](https://vitest.dev/guide/features.html#running-tests-concurrently))